pkg/apis/pxc/v1/pxc_types.go

```diff
@@ -643,6 +643,11 @@ func (cr *PerconaXtraDBCluster) CheckNSetDefaults(serverVersion *version.ServerV
 
 	cr.setSecurityContext()
 
+	if cr.Spec.EnableCRValidationWebhook == nil {
+		falseVal := false
```

Comment:
why it is disabled by default? I think it should be enabled

Reply:
it was requested by Sergey
https://jira.percona.com/browse/K8SPXC-312?focusedCommentId=275863&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-275863

Reply:
as discussed in this release we are going to disable it by default.